src/components/TodoContent.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoList } from './TodoList';
+import {
+  deleteSelectedTodo,
+  getTodos,
+  postTodo,
+  updateTodo,
+  USER_ID,
+} from '../api/todos';
+import { Errors } from '../types/Errors';
+import classNames from 'classnames';
+import { FilterBy } from '../types/FilterBy';
+import { Header } from './Header';
+import { Footer } from './Footer';
+
+export const TodoContent: React.FC = () => {
+  const [todoValue, setTodoValue] = useState<string>('');
```

Comment:
it's redundant to specify types for primitives 
```suggestion
  const [todoValue, setTodoValue] = useState('');
```